io/snmp/snmp.html

```diff
@@ -35,33 +34,91 @@
+    RED.nodes.registerType('snmp', {
         category: 'network-input',
-        color:"YellowGreen",
+        color: "YellowGreen",
         defaults: {
-            host: {value:"127.0.0.1"},
-            community: {value:"public"},
-            version: {value:"1",required:true},
-            oids: {value:""},
-            name: {value:""}
+            host: { value: "127.0.0.1" },
+            community: { value: "public" },
+            version: { value: "1", required: true },
+            oids: { value: "" },
+            name: { value: "" }
         },
-        inputs:1,
-        outputs:1,
+        inputs: 1,
+        outputs: 1,
         icon: "snmp.png",
-        label: function() {
-            return this.name||"snmp "+this.host;
+        label: function () {
+            return this.name || "snmp " + this.host;
         },
-        labelStyle: function() {
-            return this.name?"node_label_italic":"";
+        labelStyle: function () {
+            return this.name ? "node_label_italic" : "";
         }
     });
+
+</script>
+
+<script type="text/x-red" data-template-name="snmp set">
+    <div class="form-row">
+        <label for="node-input-host"><i class="fa fa-globe"></i> Host</label>
+        <input type="text" id="node-input-host" placeholder="localhost">
+    </div>
+    <div class="form-row">
+        <label for="node-input-community"><i class="fa fa-user"></i> Community</label>
+        <input type="text" id="node-input-community" placeholder="public">
+    </div>
+    <div class="form-row">
+        <label for="node-input-version"><i class="fa fa-bookmark"></i> Version</label>
+        <select type="text" id="node-input-version" style="width: 150px;">
+            <option value="1">v1</option>
+            <option value="2c">v2c</option>
+        </select>
+    </div>
+    <div class="form-row">
+        <label for="node-input-varbinds"><i class="fa fa-tags"></i> Varbinds</label>
+        <textarea rows="10" cols="60" id="node-input-varbinds" placeholder="e.g. [ { &quot;oid&quot;: &quot;1.3.6.1.2.1.1.5.0&quot;,&quot;type&quot;: &quot;OctetString&quot;,&quot;value&quot;: &quot;host1&quot;},{&quot;oid&quot;: &quot;1.3.6.1.2.1.1.6.0&quot;,&quot;type&quot;: &quot;OctetString&quot;,value: &quot;somewhere&quot;}]" style="width:70%;"></textarea>
+    </div>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> Name</label>
+        <input type="text" id="node-input-name" placeholder="Name">
+    </div>
 </script>
 
+<script type="text/x-red" data-help-name="snmp set">
+    <p>Simple snmp Set node. Trigger by any input</p>
+    <p><code>msg.host</code> may contain the host.</p>
+    <p><code>msg.community</code> may contain the community.</p>
+    <p><code>msg.varbinds</code> may contain varbinds as an array of json objects containing multiple oids, types and values.<code>[ { "oid": "1.3.6.1.2.1.1.5.0","type": "OctetString", "value": "host1"} , { "oid": ...} ]</code></p>
```

Comment:
ok, to make it all at once, what do you think in placeholder area ? I can do line breaks (&10;), but not sure about tabs.